zone/lua_entity_list.cpp

```diff
@@ -375,8 +375,19 @@ Lua_Bot_List Lua_EntityList::GetBotList() {
 	Lua_Safe_Call_Class(Lua_Bot_List);
 	Lua_Bot_List ret;
 	auto &bot_list = self->GetBotList();
-	for (auto bot_iterator : bot_list) {
-		ret.entries.push_back(Lua_Bot(bot_iterator));
+	for (auto bot : bot_list) {
+		ret.entries.push_back(Lua_Bot(bot));
+	}
+
+	return ret;
+}
+
+Lua_Bot_List Lua_EntityList::GetBotListByClientName(std::string client_name) {
+	Lua_Safe_Call_Class(Lua_Bot_List);
+	Lua_Bot_List ret;
+	auto &bot_list = self->GetBotListByClientName(client_name);
```

Comment:
error: cannot bind non-const lvalue reference of type 'std::vector<Bot*>&' to an rvalue of type 'std::vector<Bot*>'

Reply:
Yeah I saw, I’ll get it when at PC.